nondocker-internal-service-image/README.md

```diff
@@ -0,0 +1,6 @@
+# hosting
```

Comment:
Could you replace this line with `nondocker-internal-service-image`?

Reply:
oops forgot to change that 😆